ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -285,8 +285,8 @@ tests:
   postsubmit: true
   steps:
     env:
-      DESTINATION_BRANCH: community-4.20
-      SOURCE_BRANCH: release-4.20
+      DESTINATION_BRANCH: community-4.21
```

Comment:
I'm fine with this change (it's what it is) but the `community-4.21` branch does not exist 😢